metadata.json

```diff
@@ -139,7 +139,7 @@
       "orcid": "0000-0002-0735-5184"
     }
   ],
-  "description": "Networks of species interactions can capture meaningful information on the structure and functioning of ecosystems. Yet the scarcity of existing data, and the difficulty associated with comprehensively sampling interactions between species, means that to describe the structure, variation, and change of ecological networks, we need to rely on modelling tools. These tools need to have the capacity to make accurate predictions about how species interact over both time and space. Here we provide a proof-of-concept, where we show how a simple neural-network model makes accurate predictions about species interactions. We then use this model to reconstruct a metaweb of host-parasite interactions across space, and assess the challenges and opportunities associated with improving interaction predictions. Finally, we provide a primer on the relevant methods and tools that will guide the path forward for the development and integration of multiple sources of data and methodological approaches (including statistical, dynamical, and inferential models).",  
+  "description": "Networks of species interactions underpin numerous ecosystem processes, but comprehensively sampling these interactions is difficult. Interactions intrinsically vary across space and time, and given the number of species that compose ecological communities, it can be tough to distinguish between a true negative (where two species never interact) from a false negative (where two species have not been observed interacting even though they actually do). Assessing how likely interactions between species is a necessary imperative for several fields of ecology. This means that to predict interactions between species---and to describe the structure, variation, and change of the ecological networks they form---we need to rely on modeling tools. Here we provide a proof-of-concept, where we show a simple neural-network model makes accurate predictions about species interactions given limited data. We then  assess the challenges and opportunities associated with improving interaction predictions, and provide a conceptual roadmap forward toward predictive models of ecological networks that is explicitly spatial and temporal. We conclude with a brief primer on the relevant methods and tools needed to start building these models, which we hope will guide this research program forward.",  
```

Comment:
Assessing how likely interactions between species is a necessary imperative for several fields -> Assessing **the likelihood of** interactions between species is a necessary ~~imperative~~ for several fields